sympy/physics/mechanics/functions.py

```diff
@@ -532,7 +539,7 @@ def _recurser(expr, sub_dict):
 
 def _fraction_decomp(expr):
     """Return num, den such that expr = num/den"""
-    if not isinstance(expr, Mul):
+    if not isinstance(expr, MulClass):
```

Comment:
Yeah, looking into `sympy.core.backend` it looks like `Mul` and `MulClass` are exactly the same thing, but in SymPy and SymEngine. @isuruf will have to answer this one.